### PR TITLE
Added lux to accessoryType 'custom'.

### PR DESCRIPTION
**Pull request by banboobee**
Dear @simont77,

This PR adds light-level(lux) entry to accessoryType 'custom' based on @ebaauw's comment https://github.com/simont77/fakegato-history/issues/109#issuecomment-1149053372. This is working over a year, and I hope to be integrated in this repository.

Thanks in advance.

### CODE REVIEW

File: fakegato-history.js
Comment (simont77) [inline fakegato-history.js:1]: Could you also update comment on line 542 (averager works also on Lux)?

Comment (banboobee) [reply fakegato-history.js:1]: Thanks prompt reply.

Yes, lux entries are averaged, and I described in README.md as follows,

> line 110: Temperature, Humidity, Pressure, Power and **lux** entries are averaged over the history interval.

But I'm not sure that this is the line 542 in your comment.

Am I misunderstanding?

Comment (banboobee) [reply fakegato-history.js:1]: Sorry, I understood. You mean line 542 of fakegato-history.js. OK, I will.

Comment (banboobee) [reply fakegato-history.js:1]: I did.

**Comment by simont77** [comment]
Merged

**Comment by banboobee** [comment]
@simont77,

Thank you.

**Pull request merged**